### ISSUE TITLE
Badge symbol as icon ?

### ISSUE THREAD

**Issue by GaryOlsson** [game-icons/icons#821]
Hello there, I have a small suggestion to make :).

In many RPG, you have some basic symbol as quest/event indicator. For example in word of warcraft the question mark and exclamation mark are used for quest. Exclamation mark is also used in Hades for certain room. There's probably more example but you get the idea.

Since those symbols already exist as badge, it would be great if we could have the same symbol as icons but bigger (the size of badge is limited in the studio)  and without the with circle around them. 

It may not be the good spot for that but I would also like to thanks all contributor, the project is truly awesome, thank you for all the amazing icons !

**Comment by TheBl4ckSh33p** [comment]
Yes, please! I would love to have badges as full sized icons (like exclamation or info for quests).
